cms/djangoapps/contentstore/features/video.py

```diff
@@ -1,67 +0,0 @@
-# pylint: disable=missing-docstring
```

Comment:
Doesn't this test creating a video component? Don't we still need a test for that?